content/de/verein/event/fnf-at-39c3-workshop/index.md

```diff
@@ -0,0 +1,21 @@
+{
+    "title": "Datenschutzrechte zum Mitmachen – DSGVO-Anfragen stellen mit Datenanfragen.de",
+    "type": "event",
+    "event_date": "2025-12-28T14:00:00+02:00",
+    "duration": "60m",
+    "place": {
+        "name": "39c3, Freedom-not-Fear-Assembly"
+    },
+    "license": "cc-by-40",
+    "description": "Die DSGVO ist in aller Munde, aber was bedeutet sie eigentlich für Dich? Wir wollen uns die Rechte, die Dir durch die DSGVO zugesichert werden, einmal genauer anschauen und – vor allem — ausprobieren. In unserem Workshop lernst Du, mit Datenanfragen.de Anfragen zu stellen, um an Deine Daten zu kommen, sie löschen zu lassen, zu berichtigen oder Widerspruch gegen Direktwerbung einzulegen."
+}
+
+Online-Shops, Streamingdienste, Banken, Handy-Apps, öffentliche Stellen, Versicherungen, soziale Netzwerke … Die Liste an Organisationen, die Deine Daten haben wollen, ist sehr lang. Wahrscheinlich gibt es sogar etliche Unternehmen wie Adresshändler und Auskunfteien, die zwar Deine Daten haben, von denen Du aber noch nie etwas gehört hast.
+
+Zum Glück musst Du Dich nicht damit abfinden, „dass das halt so ist“. Denn Du hast ein Recht auf Datenschutz! Wenn Unternehmen, Behörden und andere Organisationen Deine personenbezogenen Daten nutzen wollen, müssen sie sich dabei an die Datenschutz-Grundverordnung (DSGVO) halten. Damit Du die Kontrolle über Deine Daten behalten kannst, gewährt Dir die DSGVO eine Reihe von Rechten, die Du den Organisationen gegenüber kostenlos geltendmachen darfst: Das Recht auf Auskunft darüber, welche Daten zu Dir verarbeitet werden, einschließlich einer Kopie dieser Daten, das Recht auf Berichtigung falscher Daten zu Dir, das Recht auf Löschung Deiner Daten unter bestimmten Umständen, und das Recht auf Werbewiderspruch.
+
+Wir sind ein gemeinnütziger Verein, der daran arbeitet, diese Rechte allen Menschen zugänglich zu machen. Dafür betreiben wir vor allem die Webseite Datenanfragen.de, die Dich durch den gesamten Anfragenprozess leitet und am Ende den Anfragentext generiert, den Du nur noch an die entsprechenden Organisationen abschicken musst.
+
+In diesem Workshop wollen wir Dich einladen, von Deinen Rechten Gebrauch zu machen und mehr darüber zu erfahren, wer welche Daten zu Dir hat. Dazu erklären wir Dir näher, was es mit Deinen Datenschutzrechten und unseren Werkzeugen auf sich hat und stellen dann gemeinsam Anfragen. Keine Sorge: Du brauchst dafür keine technischen oder rechtlichen Vorkenntnisse. Der Workshop richtet sich vor allem an Menschen, die bisher keine oder wenig Erfahrung mit DSGVO-Anfragen haben. Alle anderen sind aber natürlich auch willkommen.
+
+**Material**: [Vortragsfolien](https://static.dacdn.de/talks/slides/2025-12-28-fnf-at-39c3-workshop.pdf)
```

Comment:
- The link to DAZN on p.15 is a 404
- the matrix link is styled as a link but not clickable